frontend/src/utils/useImageStorage/useImageStorage.ts

```diff
@@ -0,0 +1,154 @@
+import { useCallback, useState } from 'react';
+import { getStorageItem, setStorageItem, STORAGE_KEYS } from '../storage';
+
+const MAX_LOCAL_STORAGE_BYTES = 4 * 1024 * 1024;
+
+export type StoredImage = {
+  id: string;
+  dataUrl: string;
+};
+
+/**
+ * Custom hook for managing image storage in localStorage.
+ * @returns {object} - The image storage methods and error state.
+ */
+export const useImageStorage = () => {
```

Comment:
could you use the `const useImageStorage = () => {}` and `export default useImageStorage` approach that we do in other files?